packages/desktop-client/src/components/sidebar/Accounts.tsx

```diff
@@ -43,15 +37,17 @@ export function Accounts({
     (state: State) => state.account.accountsSyncing,
   );
 
-  const getAccountPath = account => `/accounts/${account.id}`;
+  const getAccountPath = (account: AccountEntity) => `/accounts/${account.id}`;
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏thanks!

Reply:
Thanks, I "typed" the last couple ts errors and removed @ts-strict-ignore from this file.